.github/workflows/codeql.yml

```diff
@@ -0,0 +1,59 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main, 'release-*' ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main, 'release-*' ]
+  schedule:
+    # run at 9 AM IST, every day.
+    - cron: '30 3 * * *'
+
+permissions:
+  actions: read
+  contents: read
+  security-events: write
```

Comment:
Fixes #889

Reply:
Looks like I'd mistaken the issue for not being able to report under the security tab as the link was unavailable..